tardis/montecarlo/montecarlo_numba/formal_integral.py

```diff
@@ -743,12 +743,24 @@ def trapezoid_integration(array, h):
 
 
 @njit(**njit_dict_no_parallel)
-def intensity_black_body(nu, T):
-    """Get the black body intensity at frequency nu
-    and temperature T"""
+def intensity_black_body(nu, temperature):
+    """
+    Read black body intensity.
```

Comment:
The previous one line looks good for the summary, you can actually keep that.

Reply:
https://tardis-sn.github.io/tardis/contributing/development/code_quality.html#docstrings
but it is saying that it is in one line and does not contain the parameter's name.

Reply:
Then maybe something like "calculate the blackbody intensity"? When it says read, it might not be obvious that this function calculates it and does not take it from some variable that already exists.